.github/workflows/go.yaml

```diff
@@ -8,22 +8,52 @@ on:
       - "go/**"
       - ".github/workflows/go.yaml"
       - "install_tooling.sh"
+  workflow_call:
+    secrets:
+      IMAGE_PREFIX:
+        description: "registry.com/loftsh in registry.com/loftsh/language:TAG"
+        required: true
 
 env:
-  IMAGE_PREFIX: loftsh
+  IMAGE_PREFIX: ${{ secrets.IMAGE_PREFIX }}
+  LANGUAGE: "go"
+  SNYK_TOKEN: ${{ secrets.SNYK_TOKEN }}
+
+defaults:
+  run:
+    working-directory: go
 
 jobs:
   go:
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - tag: 1.18-rc-alpine
+            target_tag: 1.18-rc-alpine
```

Comment:
Same here.. this is too much. Only stable images and in one flavor/distro. No RCs or betas etc.

Reply:
all rc and beta versions removed